### DISCUSSION TITLE
Examples of (for-profit) Open Source Businesses

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/diginomics#55] [General]
Examples of for-profit open source businesses current or historic. preferable successful.

- cal.com to calendly
- baserow alternative to airtable
- n8n_io to zapier
- jitsu.com alternative to segment

## cal.com - 2023-03-25

https://cal.com

![image](https://user-images.githubusercontent.com/180658/227669612-24708109-4eb4-45bb-a238-6d1f34d66d8e.png)

https://cal.com/platform

![image](https://user-images.githubusercontent.com/180658/227671605-1d50d06f-e35b-4522-b877-efbad031fda1.png)

**Comment by davidgasquez** [comment]
In the world of Data Engineering, there have been quite a few succesfull Open Source business. Sharing a few on top of my mind:

- [Snowplow Analytics](https://snowplowanalytics.com/), [Jitsu](https://jitsu.com/), and [Rudderstack](https://www.rudderstack.com/) on tracking.
- [Metabase](https://www.metabase.com/) and [Lightdash](https://www.lightdash.com/) on BI.
- [Meltano](https://meltano.com) and [Airbyte](https://airbyte.io/) on ETL.
- [Dagster](https://www.dagster.io/) and [Prefect](https://www.prefect.io/) on Orchestration.
- [Flagsmith](https://flagsmith.com/) and [Unleash](https://www.getunleash.io/) on feature flagging.


Some others outside of data:
- [Supabase](https://supabase.io/)
- [Sentry ](https://sentry.io/)
- More on _[Awesome OSS Alternatives](https://github.com/RunaCapital/awesome-oss-alternatives)_!

**Reply by rufuspollock** [reply]
Great list! Is Supabase actually open source or *built* on open source components?

**Reply by rufuspollock** [reply]
It would also be interesting to clarify the model e.g. ...

- Pure SaaS: ie. the community edition and their product are exactly the same, they just offer SaaS
- Open Core: they have an open source core (community edition) and a proprietary version with more features
- Services: they sell other services etc

**Reply by davidgasquez** [reply]
[Supabase seems to be fully open source](https://github.com/supabase/supabase).

In terms of the model... I think all of them are Open Core and with a similar approach: you can self-host but they offer a paid Cloud service.
